### DISCUSSION TITLE
Hide pre-release releases

### DISCUSSION THREAD

**Discussion by canac** [community/community#18499] [Other Feature Feedback, Questions, & Ideas]
I have some projects starred (Ionic Framework, Next.js, Remix) that release a prolific number of pre-release that clog up my feed. I'd love to have an option to hide pre-release releases and just show major/minor/bugfix releases.

**Comment by siminapasat** [comment]
@canac thank you for reporting this! We are aware of that problem (and see it on our own feeds), so we're looking at rolling up releases and showing the most recent release as the main one.

**Reply by Tiagoquix** [reply]
Hello! Any progress on this so far? Thanks in advance.

**Comment by yeelam-gordon** [comment]
Any follow up on this after like 4 years?
